sql/org_mozilla_firefox_derived/incline_executive_v1/query.sql

```diff
@@ -153,23 +166,111 @@ counts AS (
     channel,
     manufacturer,
     country
+),
+with_retention AS (
+  SELECT
+    * EXCEPT (established_churned, new_churned),
+      -- Churned users are a negative count, since they left the product
+    -1 * established_churned AS established_churned,
+    -1 * new_churned AS new_churned,
+    SAFE_DIVIDE((established_returning + new_returning), active_previous) AS retention_rate,
+    SAFE_DIVIDE(
+      established_returning,
+      (established_returning + established_churned)
+    ) AS established_returning_retention_rate,
+    SAFE_DIVIDE(new_returning, (new_returning + new_churned)) AS new_returning_retention_rate,
+    SAFE_DIVIDE((established_churned + new_churned), active_previous) AS churn_rate,
+    SAFE_DIVIDE(resurrected, active_current) AS perc_of_active_resurrected,
+    SAFE_DIVIDE(new_users, active_current) AS perc_of_active_new,
+    SAFE_DIVIDE(established_returning, active_current) AS perc_of_active_established_returning,
+    SAFE_DIVIDE(new_returning, active_current) AS perc_of_active_new_returning,
+    SAFE_DIVIDE((new_users + resurrected), (established_churned + new_churned)) AS quick_ratio,
+  FROM
+    counts
+),
+_current AS (
+  SELECT
+    *
+  FROM
+    with_retention
+  WHERE
+    date = @submission_date
+),
+last_week AS (
+  SELECT
+    * EXCEPT (date)
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_firefox_derived.incline_executive_v1`
+  WHERE
+    date = DATE_SUB(@submission_date, INTERVAL 1 WEEK)
+),
+last_year AS (
+  SELECT
+    a.date,
+    a.is_migrated,
+    a.app_name,
+    a.channel,
+    a.manufacturer,
+    a.country,
+    a.established_returning_retention_rate - b.established_returning_retention_rate AS established_returning_retention_delta
+  FROM
+    with_retention a
+  INNER JOIN
+    with_retention b
+  ON
+    a.date = DATE_ADD(b.date, INTERVAL 7 DAY)
```

Comment:
I think we use DATE_SUB everwhere else, so I'd prefer to see this also expressed that way. Also, we should use `1 WEEK` over `7 DAY` consistently.